doc/texture_packs.txt

```diff
@@ -168,3 +168,44 @@ Here are face selectors you can choose from:
 | bottom        | y-                                                |
 | sides         | x-, x+, z-, z+                                    |
 | all           | All faces. You can also use '*' instead of 'all'. |
+
+Designing leaves textures for the leaves rendering options
+----------------------------------------------------------
+
+Minetest has three modes for rendering leaves nodes if the node has the
+`allfaces_optional` drawtype.
+
+### Fancy
+
+Uses the texture specified in the `tiles` nodedef field.
+The texture should have some transparent pixels and be in the RGBA format so
+that the transparent pixels can have color information.
+Faces of every leaves node are rendered even if they are inside a solid volume
+of leaves, this gives a dense appearence.
+
+### Opaque
+
+Uses the texture specified in `tiles` but makes it opaque by converting each
+transparent pixel into an opaque pixel that uses the color information of that
+transparent pixel.
+Due to this the `tiles` texture format must be RGBA not 'indexed alpha' to allow
+each transparent pixel to have color information.
+
+The colors of the transparent pixels should be set for a good appearence in
+`opaque` mode. This can be done by painting the pixels the desired colors then
+erasing them. Then when saving the texture, use the 'save color information from
+transparent pixels' option (or equivalent).
+
+### Simple
+
+Uses the texture specified in the `special_tiles` nodedef field if it exists, if
+not, the `tiles` texture.
+The `special_tiles` texture should have some transparent pixels and be in the
+'indexed alpha' format.
+
+This mode is between the other two in terms of appearence and rendering load.
+The nodes are rendered using the `glasslike` drawtype, only showing the surface
+faces for any solid volume of leaves, not the internal faces. Due to this the
+appearence can, depending on the texture used, seem lacking in density. So the
+`special_tiles` texture option is there to provide an alternative texture with
+fewer transparent pixels for a denser appearence.
```

Comment:
Simplification: 
`special_tiles` texture provides an alternative texture with
fewer transparent pixels for a denser appearance.

Note the spelling change "appearence" to "appearance". (This also needs correcting elsewhere.)